yarn.lock

```diff
@@ -13301,6 +13298,30 @@ react-native-tab-view@^2.9.0:
   resolved "https://registry.yarnpkg.com/react-native-tab-view/-/react-native-tab-view-2.10.0.tgz#5e249e5650502010013449ffd4e5edc18a95364b"
   integrity sha512-qgexVz5eO4yaFjdkmn/sURXgVvaBo6pZD/q1eoca96SbPVbaH3WzVhF3bRUfeTHwZkXwznFTpS3JURqIFU8vQA==
 
+react-native-unimodules@~0.7.0, react-native-unimodules@~0.7.0-rc.3:
```

Comment:
Hmm, this happens when I check out a clean master, run submodule pull, and then run `yarn`.

```
git checkout master
git pull
git submodule update
yarn
```

Reply:
Because we include react-native and react-native-unimodules as Yarn workspace packages, they should never show up in yarn.lock (unless something changed with Yarn recently). When they appear in yarn.lock it usually means one of two things: the submodules aren’t checked out so Yarn can’t find the linked workspace packages, or there are packages in the repo that depend on versions of react-native or react-native-unimodules that aren’t satisfied by the workspace packages (we check for this after running yarn and print a clear warning, so I doubt that’s the case here).